transactions/hybrid-custody/send_child_nfts_with_parent.cdc

```diff
@@ -0,0 +1,53 @@
+import "NonFungibleToken"
+import "MetadataViews"
+import "ExampleNFT"
+
+import "HybridCustody"
+import "FungibleTokenMetadataViews"
+
+transaction(ids: [UInt64], to: Address, child: Address) {
+
+    // reference to the child account's Collection Provider that holds the NFT being transferred
+    let provider: auth(NonFungibleToken.Withdraw) &{NonFungibleToken.Provider}
+    let collectionData: MetadataViews.NFTCollectionData
+
+    // signer is the parent account
+    prepare(signer: auth(Storage) &Account) {
+        // get the manager resource and borrow childAccount
+        let m = signer.storage.borrow<auth(HybridCustody.Manage) &HybridCustody.Manager>(from: HybridCustody.ManagerStoragePath)
+            ?? panic("manager does not exist")
+        let childAcct = m.borrowAccount(addr: child) ?? panic("child account not found")
+        
+        self.collectionData = ExampleNFT.resolveContractView(resourceType: nil, viewType: Type<MetadataViews.NFTCollectionData>()) as! MetadataViews.NFTCollectionData?
+            ?? panic("Could not get the vault data view for ExampleNFT")
```

Comment:
You can be more generic here and borrow against the `NonFungibleToken` contract if you want

Reply:
True, I'll update this and the ft transaction to be generic before merging.